src/test/java/org/apache/datasketches/theta/CompactSketchTest.java

```diff
@@ -220,81 +221,226 @@ public void checkMemTooSmallOrdered() {
 
   @Test
   public void checkCompactCachePart() {
-    //phony values except for curCount = 0.
+    //phoney values except for curCount = 0.
```

Comment:
We can switch to UK spellings but why only this one place in an internal comment?

Reply:
Funny, when I did my spellcheck it suggested "phoney"  I didn't give it a second glance and realize that this is the British spelling :(